src/main/java/io/jenkins/pluginhealth/scoring/service/PluginService.java

```diff
@@ -56,4 +56,21 @@ public Plugin saveOrUpdate(Plugin plugin) {
     public Stream<Plugin> streamAll() {
         return pluginRepository.findAll().stream();
     }
+
+    @Transactional
+    public int getPluginsCount() {
+        return pluginRepository.findAll().size();
+    }
+
+    @Transactional
+    public int getProbeRawData(String probeID) {
```

Comment:
To match return type change

```suggestion
    public long getProbeRawData(String probeID) {
```